react/CloseButton.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from 'react'
+import { useCssHandles } from 'vtex.css-handles'
+import { formatIOMessage } from 'vtex.native-types'
+import { useIntl } from 'react-intl'
+
+import { useMinicartDispatch } from './MinicartContext'
+
+const CSS_HANDLES = [
+  'closeIconContainer',
+  'closeIconButton',
+  'closeButtonText',
+] as const
+
+interface CloseButtonProps {
+  Icon: React.ComponentType
+  text: string
+}
+
+const CloseButton: FC<CloseButtonProps> = props => {
+  const { Icon, text } = props
+  const dispatch = useMinicartDispatch()
+  const intl = useIntl()
+  const { handles } = useCssHandles(CSS_HANDLES)
+
+  const handleClick = () => {
+    dispatch({ type: 'CLOSE_MINICART' })
+  }
+
+  return (
+    <div className={`${handles.closeIconContainer} `}>
+      <button
+        className={`${handles.closeIconButton} bg-transparent pointer bg-transparent transparent bn pointer`}
```

Comment:
I would remove all the tokens here. These classes will likely have to be overwritten using the CSS handle anyway. Remember that this will be used by every store, so we should keep the default styling as plain as possible.